spec/features/idv/proofing_components_spec.rb

```diff
@@ -0,0 +1,52 @@
+require 'rails_helper'
+
+RSpec.describe 'proofing components' do
+  include DocAuthHelper
+  include IdvHelper
+  include SamlAuthHelper
+
+  let(:email) { 'test@test.com' }
+  let(:user) { User.find_with_email(email) }
+
+  before do
+    allow(IdentityConfig.store).to receive(:ruby_workers_idv_enabled).
+      and_return(ruby_workers_idv_enabled)
+    allow(IdentityConfig.store).to receive(:doc_auth_enable_presigned_s3_urls).
+      and_return(doc_auth_enable_presigned_s3_urls)
+
+    visit_idp_from_sp_with_ial2(:oidc)
+    register_user(email)
+
+    expect(current_path).to eq idv_doc_auth_step_path(step: :welcome)
+
+    complete_all_doc_auth_steps
+    click_continue
+    expect(page).to have_current_path('/verify/review', wait: 5)
+    fill_in 'Password', with: Features::SessionHelper::VALID_PASSWORD
+    click_continue
+    click_acknowledge_personal_key
+  end
+
+  context 'async proofing', js: true do
+    let(:ruby_workers_idv_enabled) { true }
+    let(:doc_auth_enable_presigned_s3_urls) { true }
+
+    it 'records proofing components' do
+      proofing_components = user.active_profile.proofing_components
+      expect(proofing_components['document_check']).to be_present
+      expect(proofing_components['document_type']).to be_present
+    end
+  end
+
+  context 'sync proofing', js: true do
+    let(:ruby_workers_idv_enabled) { false }
+    let(:doc_auth_enable_presigned_s3_urls) { false }
+
+    it 'records proofing components' do
+      proofing_components = user.active_profile.proofing_components
+      expect(proofing_components['document_check']).to be_present
+      expect(proofing_components['document_type']).to be_present
```

Comment:
so the bug that triggered this ticket was that the `liveness_required` wasn't being recorded, so it would make sense to just include that assertion here

however, a lot of the step helpers to get this done aren't set up for `js: true` and so we'd need to add in extra `expect(page).to have_path('/something', wait: 5)` in a bunch of random places, was unsure if it was worth it